src/components/panel/panel.js

```diff
@@ -126,6 +126,9 @@ angular
  *     outside the panel to close it. Defaults to false.
  *   - `escapeToClose` - `{boolean=}`: Whether the user can press escape to
  *     close the panel. Defaults to false.
+ *   - `postClose` - `{function=}`: Function that is called after close
+ *     is done. The first parameter passed into this function is the closeEvent
+ *     and the 2nd is the current panelRef.
```

Comment:
I thought `panelRef` was first because that has to be defined and `closeEvent` is optional.

You should also mention the two events for `pressEscapeToClose` and `clickBackdropToClose` or no one will know how to use them.